lib/devise-security/models/secure_validatable.rb

```diff
@@ -48,7 +49,7 @@ def self.included(base)
 
           # extra validations
           validates :email, email: email_validation if email_validation # use rails_email_validator or similar
-          validates :password, format: { with: password_regex, message: :password_format }, if: :password_required?
+          validates :password, 'devise_security/password_complexity': password_complexity, if: :password_required?
```

Comment:
Can you fix this Rubocop warning while you're in here?

Reply:
Line length? Sure, done, but now CodeClimate is complaining there are too many LoC! 😂